--- mingw-w64-x86_64-suscan-git/analyzer/device/impl/soapysdr.c	2025-01-25 21:49:35.717363900 +0100
+++ mingw-w64-x86_64-suscan-git/analyzer/device/impl/soapysdr.c	2025-01-25 21:49:51.730183100 +0100
@@ -29,6 +29,8 @@
 #include <SoapySDR/Device.h>
 #include <SoapySDR/Modules.h>
 
+#include <sigutils/util/compat-stdlib.h>
+
 SUPRIVATE const char *g_soapysdr_module_path = NULL;
 
 struct soapysdr_discovery_ctx {
