--- mingw-w64-ucrt-x86_64-sigdigger-git/SigDigger.pro	2025-01-26 17:52:08.310337500 +0100
+++ mingw-w64-ucrt-x86_64-sigdigger-git/SigDigger.pro	2025-01-26 17:57:44.109820900 +0100
@@ -69,7 +69,7 @@
 icons.files   = icons/SigDigger.png
 INSTALLS     += desktop icons
 
-RC_ICONS = sigdigger_logo.ico
+#RC_ICONS = sigdigger_logo.ico
 
 INCLUDEPATH += $$PWD/include $$SUWIDGETS_INSTALL_HEADERS
 SOURCES += \
