--- mingw-w64-x86_64-suwidgets-git/AbstractWaterfall.cpp	2025-01-25 22:43:41.510435900 +0100
+++ mingw-w64-x86_64-suwidgets-git/AbstractWaterfall.cpp	2025-01-25 22:46:47.962370300 +0100
@@ -1930,11 +1930,7 @@
 
   for (int i = 0; i < bookmarks.size(); i++) {
     x = xFromFreq(bookmarks[i].frequency);
-#if defined(_WIN16) || defined(_WIN32) || defined(_WIN64)
-    int nameWidth = ctx.metrics->width(bookmarks[i].name);
-#else
     int nameWidth = ctx.metrics->boundingRect(bookmarks[i].name).width();
-#endif
 
     int level = 0;
     int yMin = static_cast<int>(m_FATs.size()) * ctx.metrics->height();
